### DISCUSSION TITLE
Reconsider examples provided on HGVS website

### DISCUSSION THREAD

**Discussion by reece** [HGVSnomenclature/hgvs-nomenclature#196] [HGVS presentation]
A manuscript reviewer wrote:

> 4. In my opinion, the HGVS website should provide more examples. For instance, one of the pages has this: “nucleotides upstream (5') of the ATG-translation initiation codon (start) are marked with a - (hyphen-minus) and numbered c.-1, c.-2, c.-3, etc., going further upstream.”, but it would be good to also show an example, “c.-1C>T” and possibly even illustrate it with a graphic of a toy gene (for all variant categories, not just this). The current explanations will be easily comprehensible by people in molecular or human genetics (like this reviewer), but in my experience will fly way over the heads of other professionals who occasionally do genetics.

We provide diverse examples from diverse genes/transcripts on the website. The goal of this discussion is to consider whether we should reconsider our strategy regarding examples. Specifically, should we:

- Choose a minimal set of genes/transcripts across all variant types. The ideal case, perhaps, is to find a single gene with a broad mutational spectrum that is sufficient to demonstrate all variant classes.
- Draw cartoons of the transcript, protein, and genomic context (with alignments) for the small number of transcripts used.
- Provide a mechanism to demonstrate sequence context where necessary. For example, when discussing indel shuffling, it would be helpful to provide a genome-transcript alignment and compare shuffling on the genome with shuffling on the transcript, which should not cross intron/exon boundaries.

In the discussion, please indicate level of interest in this idea. Suggestions for specific transcripts and genes might be helpful.

**Comment by reece** [comment]
Some ideas for gene and transcript selection criteria:

- Alignments to multiple primary assemblies
- Multiple transcripts, with one of the MANE
- Small-ish gene with small-ish number of exons
- CDS start in 2nd exon (to counter common assumption that it's always in the 1st exon)
- genome-transcript indel and/or mismatch in GRCh37 that was resolved in 38
- Diversity of variant types in ClinVar to use as examples

**Reply by ifokkema** [reply]
Would it be too much to then go for a gene on the reverse strand? That sometimes throws people off when 3' shifting variants, as the mapping between genome and transcript then requires re-normalization after the mapping.

**Reply by reece** [reply]
Great idea. Yes, reverse strand.

**Comment by ifokkema** [comment]
I do think we can benefit from doing this, but compared to the list of other issues on GitHub and on a list I keep separately, I don't think this would be a good return on investment. It'll take quite some time, and there will be relatively little benefit. There are still plenty of wonky things (incorrect examples on the pages or things for us to figure out, e.g., the NC(NM) issue) that IMHO should be handled first.

**Reply by reece** [reply]
What if we choose a default example gene as described above and write *new* examples using that variant? We don't have to redo all of the historical variants.

Bonus: At the same time, we collect variants for a conformation suite.

**Reply by jfjlaros** [reply]
Perhaps *DMD* would be a good candidate. It is large and has a lot of different transcripts.

We could also select a gene that overlaps with other genes for even more interesting use cases.

**Reply by ifokkema** [reply]
> What if we choose a default example gene as described above and write _new_ examples using that variant? We don't have to redo all of the historical variants.
> 
> Bonus: At the same time, we collect variants for a conformation suite.

Another possible bonus: We can remove all of the LRG examples at the same time.


> Perhaps _DMD_ would be a good candidate. It is large and has a lot of different transcripts.
> 
> We could also select a gene that overlaps with other genes for even more interesting use cases.

That actually also applies to _DMD_. It's (as you know) Johan's favorite gene, so many of the given examples are already related to the _DMD_ gene. It should, therefore, be a good choice as fewer examples will need to be added.
